com_migratetojoomla/admin/language/en-GB/com_migratetojoomla.ini

```diff
@@ -0,0 +1,32 @@
+; Joomla! Project
+; (C) 2010 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_MIGRATETOJOOMLA="Migrate To Joomla"
+COM_MIGRATETOJOOMLA_CHECK_MEDIA_CONNECTION="Check media connection"
+COM_MIGRATETOJOOMLA_DOWNLOAD_MEDIA="Download Media"
+COM_MIGRATETOJOOMLA_DOWNLOAD_MEDIA_SUCCESSFULLY="Download media successfully"
+COM_MIGRATETOJOOMLA_DOWNLOAD_MEDIA_UNSUCCESSFULLY="Download media unsuccessfully"
+COM_MIGRATETOJOOMLA_FS="File System"
+COM_MIGRATETOJOOMLA_FTP="FTP"
+COM_MIGRATETOJOOMLA_FTPS="FTPS"
+COM_MIGRATETOJOOMLA_FTP_BASE_DIR="FTP base directory"
+COM_MIGRATETOJOOMLA_FTP_HOST="FTP host"
+COM_MIGRATETOJOOMLA_FTP_PROTOCOL="FTP protocol"
+COM_MIGRATETOJOOMLA_FTP_PASSWORD="FTP password"
+COM_MIGRATETOJOOMLA_FTP_LOGIN="FTP login"
+COM_MIGRATETOJOOMLA_FTP_PORT="FTP port"
+COM_MIGRATETOJOOMLA_HTTP="Http"
```

Comment:
Make sure the language keys are in alphabetical order, but I can tell you know that already :-) Just a mistake here.